src/pages/Autolending/AutolendingWho.vue

```diff
@@ -123,7 +115,6 @@
 						</transition>
 					</div>
 					<template slot="controls">
-						<hr>
```

Comment:
The autolend settings lightboxes should have an hr line separating the title and the button area...